packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -17,14 +20,31 @@ import 'memory_service.dart';
 
 typedef chartStateListener = void Function();
 
+const String _filenamePrefix = 'memory_log_';
```

Comment:
nit: _filenamePrefix to _logFilenamePrefix

Reply:
done